.travis.yml

```diff
@@ -27,10 +29,10 @@ before_script:
   - phpenv config-add .travis.php.ini
   - composer self-update --1
   - composer install --prefer-dist
-  - composer test
```

Comment:
Why did you remove this?

Reply:
It was excluded from the `before_script` section (and added to the scripts section)